### PR TITLE
Remove in-app documentation

### PR DESCRIPTION
**Pull request by 2metres**
**User-Facing Changes**
Removes in-app documentation

**Description**
Closes FG-1084

### CODE REVIEW

File: packages/studio-base/src/components/KeyboardShortcut.help.md [hunk 1]
```diff
@@ -1,11 +0,0 @@
...
-
-- Select panels to group into a Tab panel - `⌘` `click`
-- Select all panels - `⌘` `a`
-- View panel shortcuts - `hover` `~`
-
-## Playback bar
-
-- Pause or play - `space`
-- Seek forward 100 ms - `⇢`
-- Seek backward 100 ms - `⇠`
```
Comment (jtbandes) [inline packages/studio-base/src/components/KeyboardShortcut.help.md:11]: RIP. I don't think we are losing all that much here.

File: packages/studio-base/src/components/DataSourceSidebar/help.md [hunk 2]
```diff
@@ -1 +0,0 @@
-Foxglove Studio can inspect robotics data from various [sources](https://foxglove.dev/docs/studio/connection/data-sources).
```
Comment (jtbandes) [inline packages/studio-base/src/components/DataSourceSidebar/help.md:1]: Good riddance 😀

File: packages/studio-base/src/components/DataSourceSidebar/DataSourceSidebar.tsx [hunk 3]
```diff
@@ -120,7 +119,6 @@ export default function DataSourceSidebar(props: Props): JSX.Element {
     <SidebarContent
       overflow="auto"
       title="Data source"
-      helpContent={helpContent}
```
Comment (jtbandes) [inline packages/studio-base/src/components/DataSourceSidebar/DataSourceSidebar.tsx:123]: Can you make sure this prop is also deleted from the SidebarContent component?

File: packages/studio-base/src/components/PanelExtensionAdapter/PanelExtensionAdapter.tsx [hunk 4]
```diff
@@ -553,7 +550,7 @@ function PanelExtensionAdapter(props: PanelExtensionAdapterProps): JSX.Element {
         ...style,
       }}
     >
-      <PanelToolbar helpContent={props.help} />
```
Comment (jtbandes) [inline packages/studio-base/src/components/PanelExtensionAdapter/PanelExtensionAdapter.tsx:556]: Can you make sure this prop is also deleted from the PanelToolbar component?

**Review by jtbandes — APPROVED** [review]
🔪🔪🔪
Added a couple of comments about some props that can be deleted now.

**Comment by 2metres** [comment]
I would still like to add back the icon in the sidebar but as a link to the website docs until the top nav goes live. 

How do I make a sidebar button a link?

**Comment by jtbandes** [comment]
You would have to make some significant changes to the Sidebar component and https://github.com/foxglove/studio/blob/main/packages/studio-base/src/components/Sidebar/index.tsx#L31. Not sure it's worth the effort if this is getting deleted soon?

**Pull request merged**